### DISCUSSION TITLE
Account Abstraction security and UX - Bucket for thoughts

### DISCUSSION THREAD

**Discussion by bumblefudge** [ChainAgnostic/secure-design#12] [Ideas]
There's an EIP about migrating from EOAs to smarter/abstracted wallets, and [the EIP magicians thread](https://ethereum-magicians.org/t/eip-7377-migration-transaction/15144/23) is pretty wild with corner-cases and attack vectors. Maybe once this spec is stable we could discuss guidance on a call and maybe take it as a collective deliverable to PR guidance into the Security Considerations section there?

**Comment by depatchedmode** [comment]
Excellent idea. Just pinning Yoav's summary of the trade-offs between this and Dan Finlay's AUTHUSURP proposal:
https://ethereum-magicians.org/t/eip-7377-migration-transaction/15144/19
